features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/site/public/conf/settings.js

```diff
@@ -23,7 +23,7 @@ const AppConfig = {
             // from the App when logged out from the IDP (eg: SSO logout from a different App).
             timeout: 2000, // Defines the timeout for the above periodical session status check
         },
-        docUrl: 'https://apim.docs.wso2.com/en/next/',
+        docUrl: 'https://apim.docs.wso2.com/en/latest/',
```

Comment:
I think we should put absolute version here, like `4.0.0` otherwise, if we release a product with this `docUrl` (pointing to latest) and when we release the next version like `4.0.1` APIM 4  UI will point to wrong doc site

Reply:
We currently don't have a doc space for 4.0.0 hence why latest has been added because in the latest pack none of the doc links are working